doc/user_guide.md

```diff
@@ -551,6 +551,165 @@ Finally duplicate links can exist. They have no real direction. the closest matc
 |--------|-------------------|
 | ?      | Duplicate         |
 
+## XML tracing report
+
+Often projects want to further process the results generated by OpenFastTrace to create statistics about requirements, 
+to add the results to a database or further analyze the results. OpenFastTrace therefore provides a reporter that
+exports all relevant collected information into a single XML file that can further be processed by other tools.
+
+The XML exporter is called `aspec` reporter. `aspec` in this case means augmented specobject. An `aspec` report 
+can be generated  by calling OpenFastTrace in the following way:
+
+    $ java -jar openfasttrace.jar trace -o aspac -f requirements.xml requirements
+
+OpenFastTrace needs to be executed  with the command `trace` to activate the reporter. The `aspec` report is selected 
+with that parameter `-o aspec`. `-f` allows to provide the name of the output file into which the XML report is 
+generated.
+
+The `aspec` reporter generates an entry per processed requirement reporting:
+
+* all relevant parameters of the requirement
+* information if a requirement is successfully covered by other requirements
+* information if all requirements covering a requirement are itself successfully covered
+* list of requirements covering a requirement
+* list of requirements linked by a requirement
+
+The XML output roughly has the following structure
+
+    <specdocument>
+        <specobjects doctype="arch">
+            <specobject>
+                <id>arch-my-architecture-requirement</id>
+                ...
+            </specobject>
+            ...
+        </specobjects>
+        ...
+    </specdocument>
```

Comment:
```suggestion
```xml
<specdocument>
    <specobjects doctype="arch">
        <specobject>
            <id>arch-my-architecture-requirement</id>
            ...
        </specobject>
        ...
    </specobjects>
    ...
</specdocument>
```
```

Reply:
Also please for all other XML segments.

Note that the closing "```" is missing in the suggestion because GitHub does not support this.